### PR TITLE
Add 2 approvals to CONTRIBUTING

### PR DESCRIPTION
**Pull request by josephearl**
**- Summary**

Adds info on number of approvals required on a PR. See https://github.com/netlify/netlify-cms/pull/368.

This wasn't mentioned to me when I became a contributor, so I added it to the docs.

**- A picture of a cute animal (not mandatory but encouraged)**

Cute monkey

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests. Each pull request must be approved by 2 contributors (not including yourself).
```
Comment (erquhart) [inline CONTRIBUTING.md]: I'm thinking we should add this under a separate section specifically for maintainers, as it won't apply to the majority of folks submitting PRs.

@calavera thoughts?

Comment (calavera) [inline CONTRIBUTING.md]: I agree with @erquhart.

As usual, check docker/docker for an awesome contributing guide:
https://github.com/moby/moby/blob/master/CONTRIBUTING.md

Comment (josephearl) [inline CONTRIBUTING.md]: @calavera thanks, I'll move it to it's own section

**Comment by calavera** [comment]
I'm going to close this for lack of activity. I'm working on writing a better contributing guide, I'll put a PR up this week.

**Pull request closed without merging**